src/components/StatePicker.js

```diff
@@ -14,31 +15,64 @@ const propTypes = {
     /** The label for the field */
     label: PropTypes.string,
 
-    /** A callback method that is called when the value changes and it received the selected value as an argument */
-    onChange: PropTypes.func.isRequired,
+    /** A callback method that is called when the value changes and it receives the selected value as an argument. */
+    onInputChange: PropTypes.func.isRequired,
 
     /** The value that needs to be selected */
     value: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
 
+    /** Indicates that the input is being used with the Form component */
+    isFormInput: PropTypes.bool,
+
+    /**
+    * The ID used to uniquely identify the input
+    *
+    * @param {Object} props - props passed to the input
+    * @returns {Object} - returns an Error object if isFormInput is supplied but inputID is falsey or not a string
+    */
+    inputID: props => FormUtils.validateInputIDProps(props),
+
+    /** Saves a draft of the input value when used in a form */
+    shouldSaveDraft: PropTypes.bool,
+
+    /** Callback that is called when the text input is blurred */
+    onBlur: PropTypes.func,
+
+    /** Error text to display */
+    errorText: PropTypes.string,
+
+    /** The default value of the state picker */
+    defaultValue: PropTypes.string,
+
     ...withLocalizePropTypes,
 };
 
 const defaultProps = {
     label: '',
-    value: '',
+    value: undefined,
+    errorText: '',
+    shouldSaveDraft: false,
+    isFormInput: false,
+    inputID: undefined,
+    onBlur: () => {},
```

Comment:
Hmm I see this console warning:
![warn](https://user-images.githubusercontent.com/22219519/165634389-b655911f-f8c4-4e0a-bed3-08d43be1497d.png)

Reply:
it's solved now, now tested the app in the web and no error is there and picker is working

![Screenshot 2022-04-28 at 11 14 19 AM](https://user-images.githubusercontent.com/35371050/165703625-c3a1c2dd-cf0c-4f2b-82ed-8dc315b5ebcb.png)

tested in storybook , its working fine and no console error

https://user-images.githubusercontent.com/35371050/165703647-34610f84-b8e0-4650-bf3e-0ca88c2588d7.mov